pyflakes/checker.py

```diff
@@ -1557,7 +1584,12 @@ def CALL(self, node):
                 node.func.attr == 'format'
         ):
             self._handle_string_dot_format(node)
-        self.handleChildren(node)
+
+        with contextlib.ExitStack() as stack:
+            if _is_typing(node.func, 'cast', self.scopeStack):
+                stack.enter_context(enter_annotation(self))
```

Comment:
I believe this testcase will trigger a false positive:

```python
cast(str, 'Union[Unknown]')
```

(notably, only the first argument of `cast` is a typing context)

Reply:
D'oh. Good spot, will fix.

Reply:
Fixed in 86e4e53bfbee362f044c52c72a552e07ec4ad5dd.